app/controllers/v1/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module V1
```

Comment:
Looks like you need to change namespece to `Api::V1::Users`

Reply:
Yes